pages/common/fisher.md

```diff
@@ -0,0 +1,32 @@
+# fisher
+
+> Fisher, a fish-shell plugin manager.
+> Install plugins by name or from a managed `fishfile` for bundled installs.
+
+- Install one or more plugins:
+
+`fisher {{plugin1}} {{plugin2}}`
+
+- Install from a GitHub gist:
+
+`fisher {{gist_url}}`
+
+- Install multiple plugins from the `fishfile`:
+
+`$EDITOR ~/.config/fish/fishfile; fisher`
```

Comment:
So is this editing the config file, and then running `fisher`?

Also, can you specify a custom `fishfile`?